### PR TITLE
[MoM] The liquid from the black Nether pools is now collectible and drinkable

### PR DESCRIPTION
**Pull request by Standing-Storm**
#### Summary
Mods "[MoM] The liquid from the black Nether pools is now collectible and drinkable"

#### Purpose of change

It's what the people asked for.

#### Describe the solution

Black nether pools are now a liquid source, providing `black liquid`:

> You don't know what this is but now that you've gotten a closer look it's definitely not water.  It's completely opaque and freezing cold to the touch, but when you pull your fingertip out of it, your skin is dry as a bone.

You can drink it.

You probably shouldn't drink it.

#### Testing
Liquid is collectible and drinkable.

#### Additional context

The water is:
<details>
  <summary>Spoiler</summary>
  
The black water visible when shifting masses appear during portal storms, where it's written:

> Small gashes in reality open nearby, revealing an endless expanse of black water

That's why it's bad for you but doesn't cause horrible alien diseases or shut down your metabolism or anything similar. It's from the psychoreactive realm of Nether things, not the alien biology one.
</details>

### CODE REVIEW

**Comment by Nebnis** [comment]
Maybe add a chance of the disease to also raise your nether attunement, and keep it up?

**Comment by Standing-Storm** [comment]
> Maybe add a chance of the disease to also raise your nether attunement, and keep it up?

Drinking it wouldn't really connect your mind or consciousness to the Nether at all, at least not directly. And from a Doylist perspective, it's not meant to be a potion that you drink to increase your power level.

File: data/mods/MindOverMatter/materials.json [hunk 1]
```diff
@@ -34,5 +34,25 @@
...
+  {
+    "type": "material",
+    "id": "black_liquid",
+    "name": "Black liquid",
+    "density": 2,
+    "//": "It desires to remain a liquid",
+    "specific_heat_liquid": 1,
+    "specific_heat_solid": 0.108,
+    "latent_heat": 800,
+    "freezing_point": -460,
```
[the remark this answers is not in the document]
Comment (Standing-Storm) [reply data/mods/MindOverMatter/materials.json:47]: Yes, it's intentional.

File: data/mods/MindOverMatter/items/comestibles.json [hunk 2]
```diff
@@ -203,5 +203,25 @@
...
+    "weight": "500 g",
+    "volume": "250 ml",
+    "charges": 1,
+    "calories": 0,
+    "quench": 0,
+    "symbol": "~",
+    "color": "black",
+    "phase": "liquid",
+    "flags": [ "NUTRIENT_OVERRIDE", "TRADER_AVOID" ],
+    "contamination": [ { "disease": "black_nether_water_disease", "probability": 100 } ],
```
[the remark this answers is not in the document]
Comment (Standing-Storm) [reply data/mods/MindOverMatter/items/comestibles.json:224]: Unfortunately I don't think there's any possible EoC interactions with contamination yet--there's also no "pick up item" event, so I can't have a bad feeling when the black liquid enters your inventory, and it doesn't look like there's any "take liquid from source" event either. It's a great idea but requires more infrastructure first.

**Pull request merged**